src/main/java/frc/robot/RobotContainer.java

```diff
@@ -499,6 +495,9 @@ private void configureBindings() {
     //     .and(m_driverController.a())
     //     .whileTrue(alignToPose(() ->
     // EagleUtil.getClosestLeftReefBack(drivetrain.getPose(0.25))));
+    IS_L4
```

Comment:
Why does this need `IS_L4` ?

Reply:
most likely it should be on `IS_L1` instead

Reply:
I was curious why the target of the coral matters at all when aligning to the barge.